src/Views/ListView/Lists/GenericList.vala

```diff
@@ -104,7 +105,7 @@ public abstract class Noise.GenericList : FastView {
 
         parent_wrapper.library.media_updated.connect (media_updated);
 
-        App.player.current_cleared.connect (current_cleared);
+        App.player.queue_cleared.connect (current_cleared);
```

Comment:
What's the difference between the two and how does this relate with the issues being fixed?

Reply:
Before my changes, you had two different list in `PlaybackManager` : `current` and `queue`. What I did is that I merged these two list into `queue`, that's why I renamed this signal.